1-mandatory/2-people-i-know.js

```diff
@@ -402,6 +421,25 @@ This time, I only want the full names of the people who can multitask
 */
 
 let colleaguesWhoCanMultitask = [];
+for (const freind of friends) {
+  for (const item of freind.colleagues) {
```

Comment:
In this case, what is other name that you could give item? what is item of friend.colleagues?